service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/GeneratedOrderControllerTest.java

```diff
@@ -315,8 +334,16 @@ private CaseDetails generateBlankOrderCaseDetails() {
                 .build();
         }
 
-        private CaseDetails generateCareOrderCaseDetailsWithFurtherDirections() {
-            final CaseData.CaseDataBuilder dataBuilder = generateCommonCareOrderDetails();
+        private CaseDetails generateCareOrderCaseDetailsWithoutFurtherDirections() {
+            final CaseData.CaseDataBuilder dataBuilder = generateCommonOrderDetails(CARE_ORDER, INTERIM);
+
+            return CaseDetails.builder()
+                .data(mapper.convertValue(dataBuilder.build(), new TypeReference<>() {}))
+                .build();
+        }
+
+        private CaseDetails generateCareOrderCaseDetailsWithInterimSubtype() {
```

Comment:
Very similar to 'generateCareOrderCaseDetailsWithFinalSubtype'. Can we refactor into the one method that takes in subType as param?

Reply:
yep 👍